api/scripts/verify_missing_features.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+
+"""
```

Comment:
Useful script to verify if there exist dataset.outputs that are not indexed in the features index.

Reply:
It's a one-file utility script. Right now it is a dev utility, but has the potential to be run in production if we want to occasionally sync datasets.outputs with the features index- the first step would be to check if the two are out of sync!

I don't foresee us using this anytime soon, but I'll give that refactor a try 👍🏼

Reply:
Addressed.